kobo/django/fields.py

```diff
@@ -42,11 +44,12 @@ class MyModel(models.Model):
 
 class StateEnumField(models.IntegerField):
     '''StateEnum DB encapsulation'''
-
+    widget = Select
```

Comment:
This supposed to be here?

Reply:
Fixed.

Reply:
I have restored it, since it was actually needed after all. The CI fails without it.

Reply:
Hmm. Do you know why? Is the issue CI-specific, i.e., is this only used by CI and can therefore be handled differently?

Reply:
Seems to be a standard for Django fields: https://code.djangoproject.com/ticket/3821. For its inner workings, Django requires a class variable instead of an object variable. 

CI: https://travis-ci.org/github/release-engineering/kobo/jobs/672835431

Reply:
Ah, okay. Thanks!